src/basic/cgroup-util.c

```diff
@@ -1287,6 +1287,9 @@ static const char *skip_session(const char *p) {
  * Skip user@*.service, but require it to be there.
  */
 static const char *skip_user_manager(const char *p) {
+        _cleanup_free_ char *i = NULL;
+        char *unit_name;
+        UnitNameFlags type;
```

Comment:
Not a matter, but nowadays we allow to declare variables any places.
Please declare them when used.